smother/control.py

```diff
@@ -41,7 +46,7 @@ def start(self):
 
     def save_context(self, label):
         self.data[label] = {
-            key: sorted(map(int, val.keys()))
+            self._normalize_path(key): sorted(map(int, val.keys()))
```

Comment:
One other though: if the path alias logic were applied at read time instead of write time, I think we could eliminate the `--relative-paths` option -- report data would always be an absolute path, and the `.coveragerc` file could specify how to normalize those paths (potentially to relative paths if desired).